### PR TITLE
adding tests and ignoring failing ones

### PR DESCRIPTION
**Pull request by MovGP0**

### CODE REVIEW

File: test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ public void LinearRegression()
             new LinearRegression() { Enabled = true }.Run();
         }
 
+        [Ignore("not working")]
```
Comment (Oceania2018) [inline test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: Why do you ingore this?

image

File: test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs [hunk 2]
```diff
@@ -74,6 +75,7 @@ public void NamedEntityRecognition()
             new NamedEntityRecognition() { Enabled = true }.Run();
         }
 
+        [Ignore("Not working")]
```
Comment (Oceania2018) [inline test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: image

File: test/TensorFlowNET.UnitTest/Hub/MnistModelLoaderTest.cs [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
 using System.Threading.Tasks;
-using Tensorflow;
 using Tensorflow.Hub;
 
 namespace TensorFlowNET.UnitTest.Hub
 {
     [TestClass]
     public class MnistModelLoaderTest
     {
+        [Ignore("not working")]
```
Comment (Oceania2018) [inline test/TensorFlowNET.UnitTest/Hub/MnistModelLoaderTest.cs]: image

File: test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs [hunk 4]
```diff
@@ -106,14 +106,15 @@ public void NeuralNetXor()
             Assert.IsTrue(new NeuralNetXor() { Enabled = true, IsImportingGraph = false }.Run());
         }
 
+        [Ignore("Not working")]
```
Comment (Oceania2018) [inline test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: Please remoe `Ignore`.

File: test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs [hunk 5]
```diff
@@ -106,14 +106,15 @@ public void NeuralNetXor()
...
         [TestMethod]
         public void NeuralNetXor_ImportedGraph()
         {
             tf.Graph().as_default();
             Assert.IsTrue(new NeuralNetXor() { Enabled = true, IsImportingGraph = true }.Run());
         }
 
-
+        [Ignore("Not working")]
```
Comment (Oceania2018) [inline test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: Please remove `Ignore`.

File: test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs (hunk 1 above)
Comment (MovGP0) [reply test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: those tests are all failing on my machine.

Comment (MovGP0) [reply test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: I've updated visual studio and they are working now. not sure why. maybe an issue with a file handle 🤔

updated the change request.

File: test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs (hunk 2 above)
Comment (MovGP0) [reply test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: done

File: test/TensorFlowNET.UnitTest/Hub/MnistModelLoaderTest.cs (hunk 3 above)
Comment (MovGP0) [reply test/TensorFlowNET.UnitTest/Hub/MnistModelLoaderTest.cs]: done

File: test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs (hunk 4 above)
Comment (MovGP0) [reply test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: done

File: test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs (hunk 5 above)
Comment (MovGP0) [reply test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs]: done

**Pull request merged**